tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +112,167 @@ module('Acceptance | guid file | registration files', hooks => {
             .hasAria('label', t('file_detail.view_revisions'), 'Versions button has correct label when closed');
     });
 
-    // test for tags
+    // Test file metadata
+    test('view metadata', async function(this: ThisTestContext, assert) {
+        const nodeNoun = ['Registration', 'Project', 'Component'];
+        const metadataRecord = await this.owner.lookup('service:store')
+            .findRecord('custom-item-metadata-record', this.registration.id);
+
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        // Verify correct default tab for dekstop
+        assert.dom('[data-test-metadata-tab]').exists();
+        assert.dom('[data-test-revisions-tab]').doesNotExist();
+        assert.dom('[data-test-tags-tab]').doesNotExist();
+
+        // Verify favicon
+        assert.dom('[data-test-metadata-button]').hasAria('label', 'Close metadata',
+            'Close metadata aria should be present when metadata tab is open.');
+        assert.dom('[data-test-metadata-button] > svg').hasClass('fa-info-circle',
+            'Button should have info circle Font Awesome 5 icon present.');
+
+        // File Metadata
+        assert.dom('[data-test-metadata-header]').hasText('File Metadata',
+            'File metadata header is properly set.');
+        // Title
+        assert.dom('[data-test-file-title-label]').hasText('Title',
+            'File metadata title label text is properly set.');
+        assert.dom('[data-test-file-title]').exists();
+        // Description
+        assert.dom('[data-test-file-description-label]').hasText('Description',
+            'File metadata description label text is properly set.');
+        assert.dom('[data-test-file-description]').exists();
+        // Resource type
+        assert.dom('[data-test-file-resource-type-label]').hasText('Resource type',
+            'File metadata resource type text is properly set.');
+        assert.dom('[data-test-file-resource-type]').exists();
+        // Resource language
+        assert.dom('[data-test-file-language-label]').hasText('Resource language',
+            'File metadata resource language text is properly set.');
+        assert.dom('[data-test-file-language]').exists();
+
+        // Node metadata
+        const nodeNounElement = document.querySelectorAll('[data-test-metadata-node]')[0].textContent;
+        const nounFound = nodeNoun.some(r => nodeNounElement?.includes(r));
+        assert.equal(nounFound, true, 'Node noun properly set.');
```

Comment:
A separate test for a registration node noun has now replaced the previous one. We may want to include the tests for components and projects in another ticket since from our conversation this is slightly out of scope. In the meantime, I can include my check against the node noun array for Registration, Project, and Component back if you think it helps. Let me know which you would prefer.

Reply:
No, that one is in scope. On the ticket, it was the second bullet point, "Ensure that the label for the node section correctly populates project, component, or registration. "

Reply:
Yes, it was just resolved differently. I have it checking values for each Project, Component, or Registration but understand you would like to see the test constructed for a top-level structure that navigates to each and verifies if the label states it is a Project's, Component's, or Registration's metadata. This can also be added.